packages/cli/src/commands/fix/tasks/convert-task.ts

```diff
@@ -14,14 +14,14 @@ const __dirname = dirname(__filename);
 const workerPath = resolve(__dirname, 'convertWorker.js');
 
 export function convertTask(
-  targetPath: string,
+  srcPath: string,
```

Comment:
Why are you removing targetPath?

Reply:
we arent removing it; just aligning the naming convention to the other tasks